pyproject.toml

```diff
@@ -149,6 +146,7 @@ manylinux-aarch64-image = "manylinux_2_28"
 manylinux-pypy_x86_64-image = "manylinux_2_28"
 manylinux-pypy_aarch64-image = "manylinux_2_28"
 
+enable = ["pypy"]
```

Comment:
I think we can check if those are getting any downloads, and then consider maybe to deprecate building them

I don't think lot of users is using pypy for their production

Reply:
and even if they do, they can still build it it themselves